docs/advanced/queryNsr.mdx

```diff
@@ -25,8 +25,9 @@ The GraphQL query string.
 An object with GraphQL variables.
 
 ### options (`RequestOptions`) [Optional]
+
 An object containing a subset of `RequestInit` options that's applied to the request.
 
-| Key      | Type          | Description |
-|:---------|:--------------|:------------|
+| Key      | Type          | Description                                                                                                                                       |
+| :------- | :------------ | :------------------------------------------------------------------------------------------------------------------------------------------------ |
```

Comment:
Bare lurer. Hva er dette for?

Reply:
Har berre kjørt automatisk formattering med Prettier. Kan sjekke om eg kan fjerne litt av dette.